app-misc/ledit/ledit-2.04.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A line editor to be used with interactive commands"
+SRC_URI="http://pauillac.inria.fr/~ddr/ledit/distrib/src/${P}.tgz"
+HOMEPAGE="http://pauillac.inria.fr/~ddr/ledit/"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="+ocamlopt"
+RESTRICT="installsources !ocamlopt? ( strip )"
+
+DEPEND=">=dev-lang/ocaml-3.09:=[ocamlopt?]
+	dev-ml/camlp5:="
+RDEPEND="${DEPEND}"
+
+src_compile() {
```

Comment:
Add `QA_FLAGS_IGNORED="/usr/bin/ledit"` between these two sections, and a short comment above why it can't be fixed.

Reply:
Done. Is it enough to leave a hyperlink targeting my explanation from the previous message?